### ISSUE TITLE
A test issue

### ISSUE THREAD

**Issue by farooqkz** [ivan-hc/test#1]
only for test ;)

**Comment by ivan-hc** [comment]
yes, in this way I've just to upload the right workflow when I must create an appimage to distribute, I have some issues with mpv and handbrake latelly due to the PPA's origin
